http/vertx/src/main/java/io/cloudevents/http/vertx/VertxMessageFactory.java

```diff
@@ -56,7 +56,7 @@ public static MessageReader createReader(MultiMap headers, Buffer body) throws I
     }
 
     /**
-     * Like {@link this#createReader(HttpServerRequest)}
+     * Like {@link #createReader(HttpServerRequest)}
      *
      * @param request
      * @param handler
```

Comment:
Can you document this method and its parameters and add a comment like `@see #createReader(HttpServerRequest, Handler)` to `createReader(HttpServerRequest)`